static-analysis/src/main/java/com/linecorp/lich/static_analysis/LichIssueRegistry.kt

```diff
@@ -5,14 +5,19 @@ import com.android.tools.lint.detector.api.CURRENT_API
 import com.android.tools.lint.detector.api.Issue
 import com.linecorp.lich.static_analysis.detectors.OptionalArgumentRequiredDetector
 import com.google.auto.service.AutoService
+import com.linecorp.lich.static_analysis.detectors.LichFactoryDetector
 
 /**
  * A class to register custom [Issue]s
  */
 @AutoService(IssueRegistry::class)
 class LichIssueRegistry : IssueRegistry() {
     override val issues: List<Issue>
-        get() = listOf(OptionalArgumentRequiredDetector.ISSUE)
+        get() = listOf(
```

Comment:
Since this list is static, how about changing like this?
```kotlin
    override val issues: List<Issue> = listOf(
        OptionalArgumentRequiredDetector.ISSUE,
        LichFactoryDetector.TYPE_ARGUMENT_ISSUE,
        LichFactoryDetector.OBJECT_ISSUE
    )
```